hooks/post-turn-quality-stop-hook.py

```
@@ -0,0 +1,925 @@
+    if not commands:
+        return 0
+
+    ok_all = all(int(c.get("exit_code", 0)) == 0 for c in commands)
+    if ok_all:
+        return 0
+
+    state.ok = False
+    return block_and_print(state)
+
+
+def run_stop_checks(start_cwd: Path, base_ref: str, always_fetch: bool, max_out: int) -> int:
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)